### ISSUE TITLE
Track pan slider should be disabled for instruments that doesn't support note panning

### ISSUE THREAD

**Issue by kmatheussen** [kmatheussen/radium#561]

**Comment by kmatheussen** [comment]
Maybe even remove the pan slider, and make the volume slider taller

**Comment by onukore** [comment]
or made mute and solo botton instead :)

**Comment by kmatheussen** [comment]
Not going to remove the pan slider for all instrument, just if the instrument doesn't support note panning. Another alternative is to let the pan slider control the global pan value for the instrument, like before. Maybe that's better.

**Comment by kmatheussen** [comment]
> i don't use this pan or volume , i prefert system pan and volume

Well, I'm not going to remove a feature just because you don't use it. But it would make sense in the preferences to have an option to show/hide it.

**Comment by onukore** [comment]
i don't really anderstand why we have 2 pan and 4 volume ,but if you estimated is go think, why not ...
has you witch my friend, has you witch...  ;)

**Comment by Lykdraft** [comment]
> i don't use this pan or volume , i prefert system pan and volume

I use both.. ;) I prefer the latter one like you as well, but I like the fact that the sliders in the track header  can be set **independently** for every block. _(Did you know that? )_ Not too much use for  pan, but pretty awesome for volume.

**Comment by kmatheussen** [comment]
I think track pan can be pretty awesome as well. Lets say you have a melody and you want the sound in the right channel to sound a little bit later than in the left channel. Then put the melody into two tracks, each panned in different channels, and skew right channel a little bit down compared to left channel. You can of course do it other ways too, but this is simpler I think, and you can also have slightly different melodies in the two tracks.

**Comment by Lykdraft** [comment]
> i don't really anderstand why we have 2 pan and 4 volume ,but if you estimated is go think, why not ...
has you witch my friend, has you witch... ;)

Like I said,  the topmost track volume is independent for each block, which eliminates time for volume automation, because if you want sound A a little bit softer in block 2, you can adjust it there, without the need of volume automation.

**Comment by Lykdraft** [comment]
Ah, lol, got me on the second.. :)

**Comment by onukore** [comment]
(Did you know that? )
yes i know, lot of time i use pan and volume in vst, in not a problem for me , i love radium like that

**Comment by Lykdraft** [comment]
> You can of course do it other ways too, but this is simpler 

Good idea.

**Comment by onukore** [comment]
i love automation every where, hehe ! ;)

**Comment by onukore** [comment]
if i can use only one block , i will use it , but max leng is 1024 lines

**Comment by Lykdraft** [comment]
> i love automation every where, hehe ! ;)

Don't tell me.. ;) 

![radauto2](https://cloud.githubusercontent.com/assets/18437561/15502023/a32053f6-21b1-11e6-9be0-a68bdca6fac9.jpg)

**Comment by onukore** [comment]
loooool like me

**Comment by kmatheussen** [comment]
There's no limit on 1024 lines. That must be a bug in the interface.

**Comment by onukore** [comment]
There's no limit on 1024 lines. That must be a bug in the interface.

oooooooooooooooooh my good

**Comment by Lykdraft** [comment]
I noticed a limit as well.  Let me check... 2000 is the uppermost I can do here.

Every number above is not accepted.

**Comment by onukore** [comment]
i use 64 bar because it's just a multiple , with 2000 it's to much complicated to remembers

**Comment by kmatheussen** [comment]
You can add 10000 lines at the time by pressing left shift + return:
![lines](https://cloud.githubusercontent.com/assets/1146279/15502163/942ec1c4-21b2-11e6-88ff-9fd307dad755.png)

I also see that I can't set more than 1024 lines when pressing extra_r. I guess those limitations are there to avoid adding xxxx lines by accident, which could freeze the program for a long time.

**Comment by kmatheussen** [comment]
Unfortunately, when switching to opengl gfx rendering two years ago, blocks with xxxx lines can be very slow to edit.

**Comment by onukore** [comment]
ok thank you for shift-return , now i will use it, and my song will only one block ...looool

**Comment by Lykdraft** [comment]
I tend to use 256, so the limit was never a problem for me. 
I just like to take a look at where (and if) there are limits.  

> I also see that I can't set more than 1024 lines when pressing extra_r. 

Interesting. Extra_R and entering 2000 works here.

**Comment by kmatheussen** [comment]
Yes, 2000 is correct. These are probably just arbitrary values set in a
hurry when writing the code.
This should probably cleaned up so people don't think there's any
limitations in the program. ;-)

**Comment by onukore** [comment]
cheater , lool
